packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -167,7 +167,7 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
     let sumOfPercent = 0;
     data.chartData!.map((point: IChartDataPoint, index: number) => {
       const pointData = point.data ? point.data : 0;
-      value = (pointData / total) * 100 ? (pointData / total) * 100 : 0;
+      let value = (pointData / total) * 100 ? (pointData / total) * 100 : 0;
```

Comment:
what is this change

Reply:
While trying to use this component in the storybook, it was giving Variable initialization error, since this value variable was not initialized before this, so had to change this.
![image](https://github.com/microsoft/fluentui/assets/70366079/2df0238a-6daf-4a7c-95e8-9ed3bbba9741)